scripts/make-diamond-immutable.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+
+
+update() {
+  # load env variables
+	source .env
+
+  # check if env variable "PRODUCTION" is true, otherwise deploy as staging
+	if [[ -z "$PRODUCTION" ]]; then
+		FILE_SUFFIX="staging."
+	fi
+
+  # get user-selected network from list
+  NETWORK=$(cat ./networks | gum filter --placeholder "Network...")
+
+  # define path of JSON file to get diamond address from
+	ADDRS="deployments/$NETWORK.${FILE_SUFFIX}json"
+
+  # get diamond address from path
+  DIAMOND=$(jq -r '.LiFiDiamondImmutable' $ADDRS)
+
+
+  echo "!!! ATTENTION !!!"
```

Comment:
Not required but you can use some cool `gum` styling to make this message stand out 

e.g. from the docs
```
gum style \
	--foreground 212 --border-foreground 212 --border double \
	--align center --width 50 --margin "1 2" --padding "2 4" \
	'Bubble Gum (1¢)' 'So sweet and so fresh!'
```

Reply:
applied

Reply:
applied